ipaserver/plugins/groupfas.py

```diff
@@ -68,12 +110,16 @@ def get_fasgroup_attribute(self, entry_attrs, options):
 
 
 def group_add_fas_precb(
-    self, ldap, dn, entry_attrs, attrs_list, *keys, **options
+    self, ldap, dn, entry, attrs_list, *keys, **options
 ):
     """Add fasgroup object class
     """
-    if options.get("fasgroup", False):
```

Comment:
You have to keep this for the ``fasgroup`` argument.

Reply:
Ah right I was only adding the objectclass if other fas attributes are present, but admins should be able to create FAS groups without them.